pontoon/base/models.py

```diff
@@ -1946,20 +1946,21 @@ def get_translation(self, plural_form=None):
             )[0]
             return translation.serialize()
 
-        else:
-            return {
-                'fuzzy': False,
-                'string': None,
-                'approved': False,
-                'pk': None,
-            }
+        return {
+            'fuzzy': False,
+            'string': None,
+            'approved': False,
+            'pk': None,
+        }
 
     @classmethod
-    def for_project_locale(self, project, locale, paths=None, statuses=None,
-                           search=None, exclude=None, extra=None, time=None, authors=None):
+    def for_project_locale(
+        self, project, locale, paths=None, status=None,
+        search=None, exclude_entities=None, extra=None, time=None, author=None,
```

Comment:
Note that here I'm changing some of those parameters names, to make them coherent throughout the code base.